tests/api/reports-import.php

```diff
@@ -151,4 +151,39 @@ public function test_import_params() {
 		$this->assertEquals( 'completed', $reports[0]['status'] );
 	}
 
+	/**
+	 * Test cancelling import actions.
+	 */
+	public function test_cancel_import() {
+		wp_set_current_user( $this->user );
+
+		// Populate all of the data.
+		$product = new WC_Product_Simple();
+		$product->set_name( 'Test Product' );
+		$product->set_regular_price( 25 );
+		$product->save();
+
+		$order = WC_Helper_Order::create_order( 1, $product );
+		$order->set_status( 'completed' );
+		$order->set_date_created( time() - ( 3 * DAY_IN_SECONDS ) );
+		$order->save();
+
```

Comment:
Should we verify/assert jobs have been queued prior to the POST to cancel them, and then also verify the queue is indeed empty afer?

Reply:
Added in b8e8f99.